### PR TITLE
Add external className to UniversalLink for external link.

### PR DESCRIPTION
**Pull request by iFlameing**

### CODE REVIEW

File: src/components/manage/UniversalLink/UniversalLink.jsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ const UniversalLink = ({
             : null
         }
         rel="noopener noreferrer"
-        className={className}
+        className={`${className ?? ''} external`}
```
Comment (sneridagh) [inline src/components/manage/UniversalLink/UniversalLink.jsx:103]: This is better and less error prone with `cx` from `classnames` lib

File: src/components/manage/UniversalLink/__snapshots__/UniversalLink.test.jsx.snap [hunk 2]
```diff
@@ -30,7 +30,7 @@ exports[`UniversalLink renders a UniversalLink component if no external(href) li
 
 exports[`UniversalLink renders a UniversalLink component with external link 1`] = `
 <a
-  className={null}
+  className=" external"
```
Comment (sneridagh) [inline src/components/manage/UniversalLink/__snapshots__/UniversalLink.test.jsx.snap:33]: see?

**Review by sneridagh — CHANGES_REQUESTED** [review]

File: src/components/manage/UniversalLink/UniversalLink.jsx (hunk 1 above)
Comment (iFlameing) [reply src/components/manage/UniversalLink/UniversalLink.jsx]: No, I incorporated the changes.

File: src/components/manage/UniversalLink/__snapshots__/UniversalLink.test.jsx.snap (hunk 2 above)
Comment (iFlameing) [reply src/components/manage/UniversalLink/__snapshots__/UniversalLink.test.jsx.snap]: updated the snapshot. Please review it once again.

File: src/components/theme/Widgets/__snapshots__/EmailWidget.test.js.snap [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 exports[`EmailWidget renders an email view widget component 1`] = `
 <a
-  className="metadata email widget"
+  className="external metadata email widget"
   href="mailto:foo@bar.com"
```
Comment (davisagli) [inline src/components/theme/Widgets/__snapshots__/EmailWidget.test.js.snap:6]: @sneridagh Do we want the external link icon for mailto and telephone links?

File: src/components/manage/LinksToItem/__snapshots__/LinksToItem.test.jsx.snap [hunk 4]
```diff
@@ -103,7 +103,7 @@ exports[`LinksToItem renders "links and references" view 1`] = `
             className=""
           >
             <a
-              className="source published"
+              className="external source published"
               href="http://localhost:3000/page-cucumber"
```
Comment (davisagli) [inline src/components/manage/LinksToItem/__snapshots__/LinksToItem.test.jsx.snap:107]: Why are links to localhost:3000 showing as external?

**Review by davisagli — CHANGES_REQUESTED** [review]

File: src/components/theme/Widgets/__snapshots__/EmailWidget.test.js.snap (hunk 3 above)
Comment (sneridagh) [reply src/components/theme/Widgets/__snapshots__/EmailWidget.test.js.snap:6]: I don't think so...

File: src/components/manage/LinksToItem/__snapshots__/LinksToItem.test.jsx.snap (hunk 4 above)
Comment (iFlameing) [reply src/components/manage/LinksToItem/__snapshots__/LinksToItem.test.jsx.snap:107]: @sneridagh waiting for your check result.

**Comment by sneridagh** [comment]
@iFlameing I will check this today.

**Comment by sneridagh** [comment]
@iFlameing @davisagli I've found out that the `publicURL` was not set in the Jest setup fixtures, I added it and updated all related tests, now http://localhost:3000 will be treated as internal (as it is in runtime).

File: src/components/theme/Widgets/__snapshots__/EmailWidget.test.js.snap (hunk 3 above)
Comment (sneridagh) [reply src/components/theme/Widgets/__snapshots__/EmailWidget.test.js.snap]: I fixed this.

File: src/components/manage/LinksToItem/__snapshots__/LinksToItem.test.jsx.snap (hunk 4 above)
Comment (sneridagh) [reply src/components/manage/LinksToItem/__snapshots__/LinksToItem.test.jsx.snap]: Fixed.

**Review by sneridagh — APPROVED** [review]

**Review by davisagli — APPROVED** [review]
@sneridagh LGTM too

**Pull request merged**